packages/editor/src/editor-styles.scss

```diff
@@ -9,7 +9,7 @@
  */
 
 body {
-	font-family: $editor-font;
+	font-family: $default-font !important;	// This !important is added temporary to start the conversation. It will need a core patch if we agree.
```

Comment:
If we were to decide indeed that it was time to change the vanilla font, it would have to happen both here, and in WordPress. So it would need a separate PR, which I will make if this PR gets positive feedback.